ice/transport/http/server.go

```diff
@@ -17,17 +19,42 @@ type Server struct {
 }
 
 func NewServer(cfg config.HttpServerConfig, h nhttp.Handler) *Server {
+	srv := &nhttp.Server{
+		Addr: cfg.Addr,
+	}
+	// TODO: this logic should be moved into access_log.go
+	srv.Handler = nhttp.HandlerFunc(func(w nhttp.ResponseWriter, r *nhttp.Request) {
+		tw := &TrackedWriter{w: w, status: 200}
```

Comment:
use 200 as default, otherwise it would be zero, learned from https://github.com/gorilla/handlers/pull/103